src/current/_includes/v24.1/known-limitations/udf-limitations.md

```diff
@@ -6,4 +6,5 @@
     - Views. [#87699](https://github.com/cockroachdb/cockroach/issues/87699)
 - User-defined functions cannot call themselves recursively. [#93049](https://github.com/cockroachdb/cockroach/issues/93049)
 - [Common table expressions]({% link {{ page.version.version }}/common-table-expressions.md %}) (CTE), recursive or non-recursive, are not supported in [user-defined functions]({% link {{ page.version.version }}/user-defined-functions.md %}) (UDF). That is, you cannot use a `WITH` clause in the body of a UDF. [#92961](https://github.com/cockroachdb/cockroach/issues/92961)
-- The `setval` function cannot be resolved when used inside UDF bodies. [#110860](https://github.com/cockroachdb/cockroach/issues/110860)
\ No newline at end of file
+- The `setval` function cannot be resolved when used inside UDF bodies. [#110860](https://github.com/cockroachdb/cockroach/issues/110860)
+- Casting subqueries to [`ENUM`]({% link {{ page.version.version }}/enum.md %})s in UDFs is not supported. [#108184](https://github.com/cockroachdb/cockroach/issues/108184)
```

Comment:
While the issue only mentions enum types, I believe this restriction applies to any user-defined type.